specs/draggable.spec.jsx

```diff
@@ -89,7 +73,7 @@ describe('react-draggable', function () {
           <div
             className="react-draggable"
             style={{
-              [transformKey]: 'translate(0px, 0px)'
+              'transform': 'translate(0px, 0px)'
```

Comment:
Technically this should remain - and I need to fix other parts of the test - reason being that we could (and should) add an old browser like IE9 to the suite.